MAUI/Radial-Menu/Populating-Items.md

```diff
@@ -0,0 +1,723 @@
+
+## Through radial menu items
+
+By passing a collection of [`SfRadialMenuItem`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.RadialMenu.SfRadialMenuItem.html), you can get the view of [`SfRadialMenu`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.RadialMenu.SfRadioButton.html) control. The radial menu item class provides various options to customize the items by giving custom views, font icons, and images. You can add radial menu items by hierarchy.
+
+### Text
+
+The following code snippet demonstrates how to Create a RadialMenu control with Items having the given text.
+
+{% tabs %}
+
+{% highlight xaml %}
+
+<?xml version="1.0" encoding="utf-8" ?>
+<ContentPage xmlns="http://schemas.microsoft.com/dotnet/2021/maui"
+             xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
+             xmlns:local="clr-namespace:RadialSample"
+             xmlns:syncfusion="clr-namespace:Syncfusion.Maui.RadialMenu;assembly=Syncfusion.Maui.RadialMenu"
+             x:Class="RadialSample.MainPage">
+    <syncfusion:SfRadialMenu>
+        <syncfusion:SfRadialMenu.Items>
+            <syncfusion:SfRadialMenuItem Text="Bold" FontSize="12"/>
+            <syncfusion:SfRadialMenuItem Text="Copy" FontSize="12"/>
+            <syncfusion:SfRadialMenuItem Text="Undo" FontSize="12"/>
+            <syncfusion:SfRadialMenuItem Text="Paste" FontSize="12"/>
+            <syncfusion:SfRadialMenuItem Text="Color" FontSize="12"/>
+        </syncfusion:SfRadialMenu.Items>
+    </syncfusion:SfRadialMenu>
+</ContentPage>
+
+{% endhighlight %}
+
+{% highlight c# %}
+
+using Syncfusion.Maui.RadialMenu;
+
+namespace RadialSample
+{
+    public partial class MainPage : ContentPage
+    {
+        public MainPage()
+        {
+            InitializeComponent();
+
+            SfRadialMenu radialMenu = new SfRadialMenu();
+
+            RadialMenuItemsCollection itemCollection = new RadialMenuItemsCollection() 
+            {
+               new SfRadialMenuItem() { Text = "Bold", FontSize = 12 },
+               new SfRadialMenuItem() { Text = "Copy", FontSize = 12 },
+               new SfRadialMenuItem() { Text = "Paste", FontSize = 12 },
+               new SfRadialMenuItem() { Text = "Undo", FontSize = 12 },
+               new SfRadialMenuItem() { Text = "Color", FontSize = 12 },
+            };
+            radialMenu.Items = itemCollection;
+            this.Content = radialMenu;
+        }
+    }
+}
+
+{% endhighlight %}
+
+{% endtabs %}
+
+N> Instead of using ObservableCollection, use `RadialMenuItemsCollection` for the RadialMenu Items list and `SubMenuItemsCollection` for the list of items within each RadialMenuItem.
+
+![Display text in .NET MAUI RadialMenu](images/populating-items/maui-radialmenu-populating-only-text.png)
+
+### ImageWithText
+
+The following code snippet demonstrates how to Display images and text in the RadialMenu items of the control.
```

Comment:
The following code sample demonstrates how to Display images and text in the RadialMenu items of the control.